R/capture.R

```diff
@@ -1,51 +1,198 @@
 
```

Comment:
That is a good idea - can you create an issue for that. Right now we have other things to worry about but eventually it would be nice